pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -469,4 +471,15 @@ contract StablePhantomPool is StablePool {
     function _getMinimumBpt() internal pure override returns (uint256) {
         return _MINIMUM_BPT;
     }
+
+    function _skipBptIndex(uint256 index) internal view returns (uint256) {
+        return index < _bptIndex ? index : index + 1;
```

Comment:
When `_skipBptIndex`  is used by `_onSwapGivenIn` and `_onSwapGivenOut`,  shouldn't it be `index - 1`?

Reply:
If we use `index - 1`, we would be repeating a past value, I think we should skip the current value and replace it by the following ones